tests/test_pets.py

```diff
@@ -0,0 +1,200 @@
+import pytest
+
+from src.data.pet.category import Category
+from src.data.pet.pet import Pet
+from src.data.pet.tag import Tag
+
+
+@pytest.mark.get_pet
+def test_finding_of_existing_pet(create_pet_api):
+    api = create_pet_api
+    pet_id = 174232
+    pet = Pet(id=pet_id, category=Category(3, 'Dog'), name='Jiber', photo_urls=[
+            'https://photo1.jpg', 'https://photo2.jpg'], tags=[Tag(1, 'tag1'), Tag(2, 'tag2')], status='availabe')
+    api.delete_pet(pet_id)  # create new pet before getting
+    api.add_pet(pet)
```

Comment:
all these lines look like a pre condition, why don't you put them into a fixture?